docs/router/framework/react/guide/path-params.md

```diff
@@ -126,3 +126,213 @@ const router = createRouter({
 
 The following is the list of accepted allowed characters:
 `;` `:` `@` `&` `=` `+` `$` `,`
+
+## Optional Path Parameters
```

Comment:
The content under _Allowed Characters_, can probably be moved to the bottom of the page.